include/cantera/thermo/EEDFTwoTermApproximation.h

```diff
@@ -0,0 +1,262 @@
+/**
+ *  @file EEDFTwoTermApproximation.h EEDF Two-Term approximation solver.
+ */
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_EEDF_TWO_TERM_APPROXIMATION_H
+#define CT_EEDF_TWO_TERM_APPROXIMATION_H
+
+#include "cantera/base/ct_defs.h"
+#include "cantera/numerics/eigen_sparse.h"
+
+namespace Cantera
+{
+
+class PlasmaPhase;
+
+/**
+ *  EEDF solver options. Used internally by class EEDFTwoTermApproximation.
+ */
+class TwoTermOpt
+{
+public:
+    TwoTermOpt() = default;
+
+    double m_delta0 = 1e14; //!< Initial value of the iteration parameter
+    size_t m_maxn = 200; //!< Maximum number of iterations
+    double m_factorM = 4.0; //!< Reduction factor of error
+    size_t m_points = 150; //!< Number of points for energy grid
+    double m_rtol = 1e-5; //!< Relative tolerance of EEDF for solving Boltzmann equation
+    string m_growth = "temporal"; //!< String for the growth model (none, temporal or spatial)
+    double m_moleFractionThreshold = 0.01; //!< Threshold for species not considered in the Boltzmann solver but present in the mixture
+    string m_firstguess = "maxwell"; //!< String for EEDF first guess
+    double m_init_kTe = 2.0; //!< Initial electron mean energy
+
+}; // end of class TwoTermOpt
```

Comment:
This structure is a bit odd at the moment. There is currently no way to set any of these options and provide them to solver. If we think these are options that the user might actually need to adjust, we need to provide an interface to do so. This can be a bit of a balancing act, though -- having too many tolerances and solver parameters that are ideally never needed just creates unnecessary confusion.